uc-crux-llvm/src/UCCrux/LLVM/Classify/Types.hs

```diff
@@ -217,48 +233,27 @@ data Unfixable
     -- If the offset that was added is symbolic and not part of an argument or
     -- global variable, it\'s not clear what kind of precondition could
     -- mitigate/avoid the error.
-    AddSymbolicOffsetToArgPointer
+    AddSymbolicOffsetToInputPointer
   deriving (Eq, Ord, Show)
 
 ppUnfixable :: Unfixable -> Text
 ppUnfixable =
   \case
-    AddSymbolicOffsetToArgPointer -> "Addition of a symbolic offset to pointer in argument"
+    AddSymbolicOffsetToInputPointer ->
+      "Addition of a symbolic offset to pointer in argument, global variable, or return value of skipped function"
 
 -- | We know how we *could* fix this in theory, but haven't implemented it yet.
 data Unfixed
   = UnfixedArgPtrOffsetArg
-  | UnfixedFunctionPtrInArg
-  | -- The following would be addressed by applying "missing precondition"
-    -- heuristics to return values from skipped functions
-    UnfixedRetReadUninitialized
-  | UnfixedRetWriteUnmapped
-  | UnfixedRetCall
-  | UnfixedRetMemset
-  | UnfixedRetPtrFree
-  | UnfixedRetPtrAddOffset
-  | UnfixedRetReadBadAlignment
-  | UnfixedRetWriteBadAlignment
-  | UnfixedRetDivRemByZero
```

Comment:
@travitch You asked about recording the function name here in the review of #718 and I deferred it, just wanted to point out that these are gone with this PR!